File: Backend/Martins.Conciliacao.API/Code/Unidade/UnidadeBLL.vb
Imports Martins.Conciliacao.Model

Namespace Code.Unidade
  Public Class UnidadeBLL
    Public Function SelectUnidade() As List(Of UnidadeModel)
      Try
        Dim dal As New UnidadeDAL()
        Return dal.SelectUnidade()
      Catch ex As Exception
        Throw ex
      End Try
    End Function
    Public Sub AtivarUnidade(Unidade As UnidadeModel)
      Try
        Dim dal As New UnidadeDAL()
        dal.AtivarUnidade(Unidade)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
    Public Sub InativarUnidade(Unidade As UnidadeModel)
      Try
        Dim dal As New UnidadeDAL()
        dal.InativarUnidade(Unidade)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
    Public Sub InsertUnidade(Unidade As UnidadeModel)
      Try
        Dim dal As New UnidadeDAL()
        dal.InsertUnidade(Unidade)
      Catch ex As Exception
        Throw ex
      End Try
    End Sub
  End Class
End Namespace

File: Backend/Financeiro.ConciliacaoWebAPI/JsonWebToken/Code/Unidade/UnidadeBLL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using JsonWebToken.Model;

namespace JsonWebToken.Code.Unidade
{
  public class UnidadeBLL
  {
    public List<UnidadeModel> SelectUnidade()
    {
      try
      {
        UnidadeDAL dal = new UnidadeDAL();
        return dal.SelectUnidade();
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void AtivarUnidade(UnidadeModel Unidade)
    {
      try
      {
        UnidadeDAL dal = new UnidadeDAL();
        dal.AtivarUnidade(Unidade);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void InativarUnidade(UnidadeModel Unidade)
    {
      try
      {
        UnidadeDAL dal = new UnidadeDAL();
        dal.InativarUnidade(Unidade);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
    public void InsertUnidade(UnidadeModel Unidade)
    {
      try
      {
        UnidadeDAL dal = new UnidadeDAL();
        dal.InsertUnidade(Unidade);
      }
      catch (Exception ex)
      {
        throw ex;
      }
    }
  }
}
